server/axios_api.js

```diff
@@ -1,61 +1,31 @@
-var axios = require("axios").default;
-var HOST = "movies-tvshows-data-imdb.p.rapidapi.com";
-var API_KEY = "030611f594msh73349b7b00af6e8p16d57ejsncbf1b45a6165";
-var URL = "https://movies-tvshows-data-imdb.p.rapidapi.com/";
-
-
-
-function get_details(id) {
-  var options = {
-    method: 'GET',
-    url: URL,
-    params: {type: 'get-movie-details', imdb: id},
-    headers: {
-      'x-rapidapi-host': HOST,
-      'x-rapidapi-key': API_KEY
-    }
-  };
-  axios.request(options).then(function (response) {
-    console.log("rating:"+response.data["imdb_rating"]+"\n"+"popularity:"+response.data["popularity"]+"\n"+"vote:"+response.data["vote_count"]);
-  }).catch(function (error) {
-    console.error(error);
-  });
-
+const { default: axios } = require("axios");
+const RATINGS = {
+    RottenTomatoes: "",
+    IMDB: ""
 }
 
-
-function get_id(t){
-  var options = {
-    method: 'GET',
-    url: URL,
-    params: {type: 'get-movies-by-title', title: t},
-    headers: {
-      'x-rapidapi-host': HOST,
-      'x-rapidapi-key': API_KEY
+const getDetails = async(title) => {
```

Comment:
looks amazing. Can we change function to something like getRatings?  instead of getDetails. Function name will be more descriptive of what it is doing.